docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -1,9 +1,246 @@
++
+```bash
+hzc map put-all -n currency --json-entry currencycodes.json
+```
++
+The `-n` flag passes the map name to your cluster.
+
+Do a quick check on your cluster to make sure that your data has written successfully.
+.	Open the dashboard of the development cluster and select *Manage Cluster* > *Management Center*.
+.	Go to *Storage* > *Maps*. You'll see that your cluster has a map called `currency` with six entries. 
+
+
+[[step-6-query-map]]
+== Step 6. Query Map Data
+You can use SQL to query the data in your `currency` map.
+
+.	Start by creating a mapping to the `currency` map.
+
++
+```bash
+hzc sql "CREATE MAPPING currency (__key INT, code VARCHAR, currency VARCHAR) TYPE IMap OPTIONS('keyFormat'='int', 'valueFormat'='json-flat')"
+```
+The SQL mapping statement does a number of things:
+
+** Adds column headings for currencies and codes
+** Creates a SQL connection to the map
+** Tells Hazelcast how to serialize and deserialize the keys and values.
+
+. Try running some simple queries against the `currency` map. For example, this query returns all data in the map and orders it by the currency code.  
++
+```bash
+hzc sql "select * from currency order by code"
+```
++
+The results look like this:
+
++
+[source,shell]
+----
++------------+--------------------+--------------------+
+|__key       |code                |currency            |
++------------+--------------------+--------------------+
+|1           |CAD                 |Canadian Dollar     |
+|4           |GBP                 |Pounds Sterling     |
+|2           |INR                 |Indian Rupee        |
+|3           |MXN                 |Mexican Peso        |
+|5           |TRY                 |Turkish Lira        |
+|6           |USD                 |United States Dollar|
++------------+--------------------+--------------------+
+
+----
+
+[[step-7-automate]]
+== Step 7. Automate Actions
+When you're ready, combine the commands that you've learned about so far into a shell script and run them from the command line. 
+
+The script first writes the currency data to your development server, queries it and then switches to your production cluster to perform the same actions.
+
+. Copy the commands that you have run in the previous steps into a shell script.
++
+.myscript.sh
+[source,bash]
+----
+#!/usr/bin/bash
+hzc map put-all -n currency --json-entry currencycodes.json
+hzc sql "CREATE MAPPING currency (__key INT, code VARCHAR, currency VARCHAR) TYPE IMap OPTIONS('keyFormat'='int', 'valueFormat'='json-flat')"
+hzc sql "select * from currency order by code"
+hzc -c config_prod.yaml map put-all -n currency --json-entry currencycodes.json
+hzc -c config_prod.yaml sql "CREATE MAPPING currency (__key INT, code VARCHAR, currency VARCHAR) TYPE IMap OPTIONS('keyFormat'='int', 'valueFormat'='json-flat')"
+hzc -c config_prod.yaml sql "select * from currency order by code"
+----
+.	Save your script to `$HOME/.local/share/hz-cli`.
+. To run the script, execute the following command:
```

Comment:
Maybe add a step to give the current user permission to execute the script? Users may not be root so it's best to add it to avoid permission errors.

`chmod 777 myscript.sh`

Reply:
Updated as requested.